go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/anacrolix/log v0.8.0
 	github.com/anacrolix/torrent v1.25.1
 	github.com/aws/aws-sdk-go v1.34.21
-	github.com/blend/go-sdk v1.1.1 // indirect
+	github.com/blend/go-sdk v0.0.0-20190205012150-4a150f307fcb // indirect
```

Comment:
Is this a downgrade?

Reply:
No. It's the same version. Tag v.1.1.1 has renamed to legacy-v1.1.1.  
it's legacy-v1.1.1